### ISSUE TITLE
Moving to Github, Please Fill Out Form

### ISSUE THREAD

**Issue by tabinks** [uchicago-mobi/2015-Winter-Forum#12]
We finally got approval for private GitHub accounts for the course.  This should make things easier for us to manage.

Please fill out the following form (you do not need to use your uchicago account) and we will get everyone set up.

http://goo.gl/forms/NXICo9Vfyn

**Comment by LingduoKong** [comment]
Hi, we have a ChicagoID and a Student ID. Which should we choose to fill the form

**Comment by susanstevens** [comment]
Do we need to wait until the private github accounts are set up to create a repository for assignment 1?  Also (just to make sure I'm understanding the instructions) should we have a repository called MPCS51030-2015-Winter-Assignment1-uchicagoid with a directory inside it called HelloWorld and then our files and subdirectories inside HelloWorld?

**Comment by JRam13** [comment]
I'm going to try to answer your questions- Andrew will let me know if the information I provide is wrong. I believe you want to use you uchicago id when filling the form (mine would be "jramos" for instance). You may use your own personal github account if you already have one, or make a new one. 

For the assignments, 1 assignment = 1 repo. Susan, you are correct in that the repository should always have the same naming convention for assignments (Class-Year-Quarter-Assignment-#). Once you create your new repo, you may use the command line or Github's own client (https://mac.github.com/) to upload and commit your projects. 

The directory structure inside your repo will most likely resemble the following when done correctly:

-RepoName(Class-Year-Quarter-Assignment-#)
     -ProjectName.xcodeproj*
     -ProjectName
     -ProjectNameTest

*without this we would not be able to open your project

**Comment by aljc** [comment]
Not sure if I'm missing this in your response...do we need to wait until
the private repos are set up?

**Comment by JRam13** [comment]
I'm going to try to answer your questions- Andrew will let me know if the information I provide is wrong. I believe you want to use you uchicago id when filling the form (mine would be "jramos" for instance). ~~You may use your own personal github account if you already have one, or make a new one.~~ 

For the assignments, 1 assignment = 1 repo. Susan, you are correct in that the repository should always have the same naming convention for assignments (MPCS51030-2015-Winter-Assignment1-uchicagoid). Once you create your new repo, you may use the command line or Github's own client (https://mac.github.com/) to upload and commit your projects. 

The directory structure inside your repo will most likely resemble the following when done correctly:

* RepoName(MPCS51030-2015-Winter-Assignment1-uchicagoid)
  * ProjectName.xcodeproj*
  * ProjectName
  * ProjectNameTest

*Important. Without this we would not be able to open your project

UPDATE: I'm not sure if Andrew is suppose to email you guys with your github credentials. I guess we'll have to wait until he responds.

**Comment by JRam13** [comment]
Aljc, I'm actually not sure about this. I don't know how you get access to the private repos. My guess is that Andrew has to do something on his end or link your account somehow. Just sit tight for now.

**Comment by tabinks** [comment]
Hi.  I was waiting on a few students to fill out the form.   You will have your github private repos in a couple of minutes.  I will send out the info.  Just stand by....

**Comment by JRam13** [comment]
Andrew, do the private repos expire after the course ends? Not sure how that works.

**Comment by tabinks** [comment]
Yes, they will be deleted after the course is over.

**Comment by tabinks** [comment]
I posted a new issue where you can find the repos.
